File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/4 - 1 and 2 December 2018/03_energy_booster.py
fruit = input()
size = input()
count_sets = int(input())

price = 0

if size == "small":
    if fruit == "Watermelon":
        price = 2 * 56
    elif fruit == "Mango":
        price = 2 * 36.66
    elif fruit == "Pineapple":
        price = 2 * 42.10
    elif fruit == "Raspberry":
        price = 2 * 20
elif size == "big":
    if fruit == "Watermelon":
        price = 5 * 28.70
    elif fruit == "Mango":
        price = 5 * 19.60
    elif fruit == "Pineapple":
        price = 5 * 24.80
    elif fruit == "Raspberry":
        price = 5 * 15.20

price_sets = price * count_sets

if 400 <= price_sets <= 1000:
    price_sets -= price_sets * 0.15
elif price_sets > 1000:
    price_sets -= price_sets * 0.5

print(f"{price_sets:.2f} lv.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/08. Online Exam - 28 and 29 march 2020/03. Energy Booster.js
function energyBooster(input) {
    // Mask
    let prices = {
        "Watermelon": {'small': 56 * 2, 'big': 28.70 * 5},
        "Mango": {'small': 36.66 * 2, 'big': 19.60 * 5},
        "Pineapple": {'small': 42.10 * 2, 'big': 24.80 * 5},
        "Raspberry": {'small': 20 * 2, 'big': 15.20 * 5}
    };

    let price = prices[input.shift()][input.shift()] * Number(input.shift());

    if (price >= 400 && price <= 1000) {price *= 0.85;}
    else if (price > 1000) {price *= 0.5;}

    console.log(`${price.toFixed(2)} lv.`)
}

energyBooster(['Watermelon', 'big', 4]);  // Should return:
// 487.90 lv.

energyBooster(['Pineapple', 'small', 1]);  // Should return:
// 84.20 lv.